test/e2e/default-fixture.js

```diff
@@ -120,9 +120,23 @@ function defaultFixture(inputChainId = CHAIN_IDS.LOCALHOST) {
       BridgeController: {
         bridgeState: {
           bridgeFeatureFlags: {
-            extensionSupport: false,
-            srcNetworkAllowlist: ['0x1', '0xa', '0xe708'],
-            destNetworkAllowlist: ['0x1', '0xa', '0xe708'],
+            extensionConfig: {
+              support: false,
+              chains: {
+                '0x1': {
```

Comment:
Do the chain keys need to be in decimal format in one place and hex format in the other?

Reply:
Left it this way to minimize changes, it felt out of scope for this PR to change that all over.

@micaelae do you have any further context on why this is set up this way?

Reply:
The bridge-api represents chainIds as decimals, but the extension uses Hex. We should make them consistent eventually